src/QMCWaveFunctions/LCAO/LCAOSpinorBuilder.cpp

```diff
@@ -0,0 +1,195 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2020 QMCPACK developers.
+//
+// File developed by: Cody A. Melton, cmelton@sandia.gov, Sandia National Laboratories
+//
+// File created by: Cody A. Melton, cmelton@sandia.gov, Sandia National Laboratories
+//////////////////////////////////////////////////////////////////////////////////////
+
+#include "QMCWaveFunctions/LCAO/LCAOSpinorBuilder.h"
+#include "QMCWaveFunctions/SpinorSet.h"
+#include "OhmmsData/AttributeSet.h"
+#include "Utilities/ProgressReportEngine.h"
+#include "io/hdf_archive.h"
+#include "Message/CommOperators.h"
+
+namespace qmcplusplus
+{
+LCAOSpinorBuilder::LCAOSpinorBuilder(ParticleSet& els, ParticleSet& ions, Communicate* comm, xmlNodePtr cur)
+    : LCAOrbitalBuilder(els, ions, comm, cur)
+{
+  if (h5_path == "")
+  {
+    APP_ABORT("LCAOSpinorBuilder only works with href");
```

Comment:
use https://github.com/ye-luo/qmcpack/wiki/Error-handling
method 4. myComm is available in builders.